src/GrafanaDashboardDeploy/src/DeployTool.cs

```diff
@@ -0,0 +1,152 @@
+using Microsoft.Azure.KeyVault;
+using Microsoft.Azure.Services.AppAuthentication;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace DotNet.Grafana
+{
+    public static class DeployTool
+    {
+        public static async Task PostExportPack(GrafanaClient grafanaClient, string inputFilePath)
+        {
+            var folderIdMap = new Dictionary<string, int>();
+
+            using var sr = new StreamReader(inputFilePath);
+            using var jr = new JsonTextReader(sr);
+            var data = await JObject.LoadAsync(jr).ConfigureAwait(false);
```

Comment:
Careful with the `var` in here. The style guide is generally that we only use var when the thing on the right is "obvious", and generally that's understood to be a constructor (e.g. `var value = new MyType()`) or a cast (e.g. `var value = (MyType)obj`).  Otherwise it's hard when looking at the code to understand what's happening (especially when we foreach over this too... it's important to know what the value is)

Reply:
"style guide" Do we have this documented anywhere so we can refer to it in the future?

Reply:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md. It's the style guideline for all contributions to the dotnet repository (it's linked from the "contributing" section of dotnet/core).

It's not super detailed, but it's got a good solid list of things to do, and it's possible it needs some more bits, since it was written several C# versions ago. But this one, in particular, it item 10. :-)

Reply:
I just reread that thing for the first time in a while... boy it's out of date... the "CodeFormatter" tool hasn't worked for years... and it's in a dead repository.  The consolidation effort hasn't quite reached all the documentation parts yet, I wonder if this file will get some love during that effort.

Reply:
Cool, this is something we should link to in our new hire documentation. @ilyas1974  :)

Reply:
new hires should probably just read the entire readme of dotnet/core.  Eventually they'll get to this file (after reading https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/contributing.md)